modules/swagger-jaxrs2/src/main/java/io/swagger/v3/jaxrs2/DefaultParameterExtension.java

```diff
@@ -177,7 +179,7 @@ private boolean handleAdditionalAnnotation(List<Parameter> parameters, Annotatio
                         paramType = setter.getParameterType(0);
                     }
 
-                    for (final Annotation fieldAnnotation : setter.annotations()) {
+                    for (final Annotation fieldAnnotation : annotations(setter.getAllAnnotations())) {
```

Comment:
see also general review, we would keep as is to maintain support for jackson 2.8.x